calcom/cal.com
# feat: User Avatar and Org Logo

## What does this PR do?



This PR adds the organization logo to the user user avatars where the "lg" size was used.

-User onboarding step
<img width="691" alt="CleanShot 2023-08-10 at 20 55 43@2x" src="https://github.com/calcom/cal.com/assets/65426560/d9fc21fa-461e-4f91-b0d9-25a37c43052d">


- User public page
<img width="904" alt="CleanShot 2023-08-10 at 20 44 40@2x" src="https://github.com/calcom/cal.com/assets/65426560/4819632d-b04e-4788-86bb-654e3c57dee4">

- User profile settings
<img width="1115" alt="CleanShot 2023-08-10 at 20 44 01@2x" src="https://github.com/calcom/cal.com/assets/65426560/20c9ace7-619e-4132-98e1-f1ae77bacaa7">


Fixes # (issue)



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change





## How should this be tested?



- Create an org and add a member
- Go to that memeber's public page. The organization logo should be displayed with the avatar
- Go to the member's profile setting's page. The organization logo should be displayed with the avatar
- Go to those two pages with a normal user. The avatar should render itself

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
